easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.16-foss-2018b-Perl-5.28.0.eb

```diff
@@ -1,23 +1,27 @@
+# easybuild easyconfig
 easyblock = 'ConfigureMake'
 
 name = 'VCFtools'
 version = '0.1.16'
 versionsuffix = '-Perl-%(perlver)s'
 
 homepage = "https://vcftools.github.io"
-description = """The aim of VCFtools is to provide 
- easily accessible methods for working with complex 
+description = """The aim of VCFtools is to provide
+ easily accessible methods for working with complex
  genetic variation data in the form of VCF files."""
 
 toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'optarch': True}
 
 source_urls = ['https://github.com/%(namelower)s/%(namelower)s/releases/download/v%(version)s/']
 sources = ['%(namelower)s-%(version)s.tar.gz']
 checksums = ['dbfc774383c106b85043daa2c42568816aa6a7b4e6abc965eeea6c47dde914e3']
 
+builddependencies = [('pkg-config', '0.29.2')]
+
 dependencies = [
     ('Perl', '5.28.0'),
-    ('HTSlib', '1.9'),
+    ('tabix', '0.2.6'),
```

Comment:
@fizwit Did you mean to change this?

Reply:
@fizwit We're including `HTSlib` as a dependency here exactly because of the `tabix` it provides.

What kind of failure are you seeing when `HTSlib` is used as a dependency?

Reply:
built successfully with tabix and HTSlib

Reply:
@fizwit I'm not sure that providing *both* `tabix` and `HTSlib` as dependency for `VCFtools` makes sense...

`HTSlib` already provides `tabix` (and also `bgzip`):

```
$ module load HTSlib/1.9-foss-2018b
which tabix
$ which tabix
/prefix/software/HTSlib/1.9-foss-2018b/bin/tabix
$ which bgzip
/prefix/software/HTSlib/1.9-foss-2018b/bin/bgzip
```

See also the note at https://github.com/samtools/tabix:

```
Note: tabix and bgzip binaries are now part of the HTSlib project.
```